libbeat/processors/actions/decode_json_fields_test.go

```diff
@@ -199,6 +199,72 @@ func TestTargetRootOption(t *testing.T) {
 	assert.Equal(t, expected.String(), actual.String())
 }
 
+func TestNotJsonObjectOrArrayDepthOne(t *testing.T) {
```

Comment:
Looks like this would fit well table tests? https://github.com/golang/go/wiki/TableDrivenTests